src/components/nav/NavDrawer.vue

```diff
@@ -115,29 +99,26 @@
           </v-list-item-title>
         </v-list-item-content>
       </v-list-item>
-      <v-list-item dense>
-        <router-link to="/channel" style="font-size: 0.825rem" class="ma-auto">
-          {{ $t("views.favorites.manageFavorites") }}
-        </router-link>
-      </v-list-item>
     </v-list>
     <br>
-    <v-sheet id="bottom-bar" dense class="grey--text mt-auto mb-0">
-      <a href="https://twitter.com/holodex" title="Twitter">
-        <v-icon small color="#1DA1F2">{{ icons.mdiTwitter }}</v-icon>
-      </a>
-      <a class="mx-2" title="Support holodex (Ko-fi)" href="https://ko-fi.com/holodex">
-        <v-icon small color="#FF5E5B">M23.881 8.948c-.773-4.085-4.859-4.593-4.859-4.593H.723c-.604 0-.679.798-.679.798s-.082 7.324-.022 11.822c.164 2.424 2.586 2.672 2.586 2.672s8.267-.023 11.966-.049c2.438-.426 2.683-2.566 2.658-3.734 4.352.24 7.422-2.831 6.649-6.916zm-11.062 3.511c-1.246 1.453-4.011 3.976-4.011 3.976s-.121.119-.31.023c-.076-.057-.108-.09-.108-.09-.443-.441-3.368-3.049-4.034-3.954-.709-.965-1.041-2.7-.091-3.71.951-1.01 3.005-1.086 4.363.407 0 0 1.565-1.782 3.468-.963 1.904.82 1.832 3.011.723 4.311zm6.173.478c-.928.116-1.682.028-1.682.028V7.284h1.77s1.971.551 1.971 2.638c0 1.913-.985 2.667-2.059 3.015z</v-icon>
-      </a>
-      <span>
-        <small class="one-liner" style="font-size: 0.7rem;">© 2020 Holodex</small>
-      </span>
+    <v-sheet id="bottom-bar" dense class="d-flex justify-center justify-space-between align-center grey--text mt-auto mb-0">
+      <div>
+        <a href="https://twitter.com/holodex" title="Twitter">
+          <v-icon small color="#1DA1F2">{{ icons.mdiTwitter }}</v-icon>
+        </a>
+        <a class="mx-2" title="Support holodex (Ko-fi)" href="https://ko-fi.com/holodex">
+          <v-icon small color="#FF5E5B">M23.881 8.948c-.773-4.085-4.859-4.593-4.859-4.593H.723c-.604 0-.679.798-.679.798s-.082 7.324-.022 11.822c.164 2.424 2.586 2.672 2.586 2.672s8.267-.023 11.966-.04
+            9c2.438-.426 2.683-2.566 2.658-3.734 4.352.24 7.422-2.831 6.649-6.916zm-11.062 3.511c-1.246 1.453-4.011 3.976-4.011 3.976s-.121.119-.31.023c-.076-.057-.108-.09-.108-.09-.443-.441-3.368-3.04
+            9-4.034-3.954-.709-.965-1.041-2.7-.091-3.71.951-1.01 3.005-1.086 4.363.407 0 0 1.565-1.782 3.468-.963 1.904.82 1.832 3.011.723 4.311zm6.173.478c-.928.116-1.682.028-1.682.028V7.284h1.77s1.97
+            1.551 1.971 2.638c0 1.913-.985 2.667-2.059 3.015z</v-icon>
+        </a>
+        <span>
+          <small style="font-size: 0.7rem;">© 2020 Holodex</small>
+        </span>
+      </div>
 
-      <v-icon x-small color="grey" class="ml-auto">
-        {{ icons.mdiEarth }}
-      </v-icon>
-      <router-link to="/settings" class=" one-liner">
-        <small class="pl-1" style="font-size: 0.7rem">{{ language }}</small>
+      <router-link to="/settings">
+        <v-img :src="'/img/flags/' + language + '.png'" max-height="11" contain />
```

Comment:
appreciate the gesture but I don't really like flags coz it's a political statement that the platform has to deal with and we don't want to deal with that...

Reply:
understandable